c2cgeoportal/scripts/c2ctool.py

```diff
@@ -419,8 +419,8 @@ def step3(self):
                     "This is an optional step but it helps to have a standard project.\n"
                     "In the `create.diff` file, there are all the changes made in the create template "
                     "that are recommended to be reported in your project.\n"
-                    "An advise to be more effective: In most of the casees it concerned a file that "
-                    "you never customise, or a file that you customised in depth, then respectively "
+                    "An advise to be more effective: In most of the cases it concerned a file that "
```

Comment:
In = in

in most the cases -> in most cases ?

concerned -> concerns

customized in depth -> have heavily customized